app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.spec.tsx

```diff
@@ -33,7 +40,7 @@ describe('DocumentCaptureTroubleshootingOptions', () => {
       wrapper: wrappers.MarketingSiteContext,
     });
 
-    const links = /** @type {HTMLAnchorElement[]} */ (getAllByRole('link'));
+    const links = getAllByRole('link') as HTMLAnchorElement[];
```

Comment:
Yeah this inconsistency is necessary to appease both Testing Library and TypeScript:

- Testing Library because it intentionally doesn't provide element querying utilities to encourage testing as the users (and assistive technology) would interact with the application.
- TypeScript because it wouldn't expect [`#target`](https://github.com/18F/identity-idp/blob/ea768acd08c1ab3ca199a51192e8281f6f235797/app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.spec.tsx#L52) to exist on anything other than an anchor element.

It would be nice if there were some way to test to see that the link opens in a new tab without referencing the attribute value directly like this, so we could avoid casting the result of `getByRole`.